### DISCUSSION TITLE
How to implement redash into my website dashboard

### DISCUSSION THREAD

**Discussion by mdaffaaryandaru** [getredash/redash#6691] [Q&A]
I am a beginner, and I have been assigned a task to use Redash on my website to manage the charts within the dashboard.im using angular for client side and hapi js for server side, and i also use mongo db as a database. How do I go about it, and where should I start learning using redash?

**Comment by gaecoli** [comment]
I don't quite understand your needs. Do you want to build a redash service, or how to use redash?

**Reply by mdaffaaryandaru** [reply]
Yess that what i want to know about, build a redash service on my website

**Reply by gaecoli** [reply]
https://redash.io/help/open-source/setup

**Reply by mdaffaaryandaru** [reply]
thank you very much, but i already read the documentation already, but i still dont understand. how can i make the redash in my website

**Reply by gaecoli** [reply]
What is the specific problem you are encountering?

**Reply by mdaffaaryandaru** [reply]
oke first of all as you know im a beginner programmer, i have some website there is chart on it. and i want to use redash for controlling the chart. can i do that ?

**Reply by gaecoli** [reply]
Of course.
